udata/core/spatial/models.py

```diff
@@ -51,7 +56,15 @@ def __unicode__(self):
     __str__ = __unicode__
 
     def __html__(self):
-        return gettext(self.name) + ' <i>(' + self.code + ')</i>'
+        return '{name} <i>({code})</i>'.format(
+            name=gettext(self.name), code=self.code)
+
+    def logo_url(self, external=False):
```

Comment:
Fine to me.
I may ad this to flask-fs before publication.

I'm wondering: do we use a placeholder for logo ?